ext/dom/inner_outer_html_mixin.c

```diff
@@ -55,12 +55,9 @@ static int dom_write_smart_str(void *context, const char *buffer, int len)
 	return len;
 }
 
-/* https://w3c.github.io/DOM-Parsing/#the-innerhtml-mixin
- * and https://w3c.github.io/DOM-Parsing/#dfn-fragment-serializing-algorithm */
-zend_result dom_element_inner_html_read(dom_object *obj, zval *retval)
+/* https://w3c.github.io/DOM-Parsing/#dfn-fragment-serializing-algorithm */
```

Comment:
The documentation cannot be found here. It was moved to https://html.spec.whatwg.org/multipage/dynamic-markup-insertion.html#fragment-serializing-algorithm-steps